### ISSUE TITLE
pw-stdin not working

### ISSUE THREAD

**Issue by hephaware** [keepassxreboot/keepassxc#3325]
I'm using Kubuntu and want to log in automatically at startup. To do this I can use the following:
kwallet-query -r keepassxc kdewallet | keepassxc --pw-stdin ~/Passwords.kdbx

## Expected Behavior
KeepassXC should run with the database open

## Current Behavior
When executing the command above, I get:
QObject::startTimer: Timers cannot have negative intervals
It also ask for Password to the Database

## Possible Solution
I have seen this issue before, so I would imagine the fix should be applied again.

## Steps to Reproduce
1. Add a password entry in kwallet named keepassxc
2. Add the masterpassword in the password field
3. Use the following command to run KeePassXC with the database open:
kwallet-query -r keepassxc kdewallet | keepassxc --pw-stdin ~/Passwords.kdbx

## Debug info
KeePassXC - Version 2.4.3
Revision: 5d6ef0c

Qt 5.12.2
Debugging mode is disabled.

Operating system: Ubuntu 19.04
CPU architecture: x86_64
Kernel: linux 5.0.0-19-generic

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey

Cryptographic libraries:
 libgcrypt 1.8.4

**Comment by droidmonkey** [comment]
It's `keepassxc --pw-stdin`

**Comment by hephaware** [comment]
Sorry for that typo. I still get:
kwallet-query -r keepassxc kdewallet | keepassxc --pw-stdin ~/Passwords.kdbx 
QObject::startTimer: Timers cannot have negative intervals
Database password:

**Comment by droidmonkey** [comment]
That would indicate you aren't getting the password from the wallet or are not ending it with a newline

**Comment by hephaware** [comment]
If I only run 
`kwallet-query -r keepassxc kdewallet`
I get the password prompted in Konsole. 
What do you mean by 'ending it with a newline?. The command above is what I execute. Where do you want me to add the newline (\n)?

**Comment by hephaware** [comment]
I have also tried:
`echo "password" | keepassxc --pw-stdin ~/Passwords.kdbx `
With the same result:
```
echo "password" | keepassxc --pw-stdin ~/Passwords.kdbx
QObject::startTimer: Timers cannot have negative intervals
Database password: 
```
Ofcause where password is replaced with my real password ;-)

**Comment by droidmonkey** [comment]
How about `keepassxc --pw-stdin ~/Passwords.kdbx << "password"`

**Comment by hephaware** [comment]
Then it just give me a new line:
```
keepassxc --pw-stdin ~/Passwords.kdbx << "password"
> 
```
If I push ctrl+d I get:
```
> bash: warning: here-document at line 24 delimited by end-of-file (wanted `password')
QObject::startTimer: Timers cannot have negative intervals
Database password: 
```

**Comment by droidmonkey** [comment]
Just to be clear, KeePassXC is not showing the gui when you use the other methods and just stuck at the database password prompt. Correct?

You can try adding a newline to the password stored in the kwallet. This emulates "submitting" the input prompt.

**Comment by hephaware** [comment]
I added a new line in kwallet, which did the trick and launched keepassxc with the database open.
I ran the command in Konsole(terminal), which still displays:
```
kwallet-query -r keepassxc kdewallet | keepassxc --pw-stdin ~/Passwords.kdbx
QObject::startTimer: Timers cannot have negative intervals
Database password: 
```
But when using the command in 'autostart', KeePassXC starts with the database open and I do not see the terminal as above.

I see this as closed, as it works. Thank you for a great support... :-D
